genome_designer/main/migrations/0008_auto__add_contig.py

```diff
@@ -0,0 +1,234 @@
+# -*- coding: utf-8 -*-
```

Comment:
Even though we're going to reset migrations soon, it's good practice to only submit  a single migration. Please redo migrations so you're only adding one.